plugins/confirm.cpp

```diff
@@ -16,27 +16,28 @@
 #include "uicommon.h"
 
 #include "df/building_tradedepotst.h"
+#include "df/gamest.h"
 #include "df/general_ref.h"
 #include "df/general_ref_contained_in_itemst.h"
 #include "df/interfacest.h"
 #include "df/viewscreen_dwarfmodest.h"
-#include "df/viewscreen_jobmanagementst.h"
-#include "df/viewscreen_justicest.h"
-#include "df/viewscreen_layer_militaryst.h"
-#include "df/viewscreen_locationsst.h"
-#include "df/viewscreen_tradegoodsst.h"
+//#include "df/viewscreen_jobmanagementst.h"
+//#include "df/viewscreen_justicest.h"
+//#include "df/viewscreen_layer_militaryst.h"
+//#include "df/viewscreen_locationsst.h"
+//#include "df/viewscreen_tradegoodsst.h"
```

Comment:
These can be deleted entirely. The classes are removed from the game and are unlikely to come back soon.